### PR TITLE
added google ID for RTB

### PR DESCRIPTION
**Pull request by swebshet**
Adding google Id and new community

### CODE REVIEW

File: dspace/modules/xmlui-mirage2/src/main/webapp/themes/RTB/sitemap.xmap [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+            Define global theme variables that are used later in this
+            sitemap. Two variables are typically defined here, the theme's
+            path and name. The path is the directory name where this theme
+            is located, such as "Reference" for the reference theme. The
+            theme's name is used only for descriptive purposes to describe
+            the theme.
+        -->
+        <map:component-configurations>
+            <global-variables>
+                <theme-path>HumidTropics</theme-path>
```
Comment (alanorth) [inline dspace/modules/xmlui-mirage2/src/main/webapp/themes/RTB/sitemap.xmap]: Theme path needs to be `RTB`.

File: dspace/modules/xmlui-mirage2/src/main/webapp/themes/RTB/sitemap.xmap [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+                <theme-name>HumidTropics</theme-name>
```
Comment (alanorth) [inline dspace/modules/xmlui-mirage2/src/main/webapp/themes/RTB/sitemap.xmap]: Theme path and name need to be `RTB`.

**Review by alanorth — CHANGES_REQUESTED** [review]
The theme also needs to be added to `dspace/config/xmlui.xconf` for handle `10568/80100`.

**Comment by alanorth** [comment]
In the future, a better commit message would be: Add XMLUI theme for Roots, Tubers, and Bananas (RTB) community.

**Comment by swebshet** [comment]
Sure, I consider it

**Pull request closed without merging**